apps/web/src/views/V3Info/components/Search/index.tsx

```diff
@@ -390,15 +392,18 @@ const Search = () => {
                       <Text ml="10px" style={{ whiteSpace: 'nowrap' }}>
                         <Text>{`${p.token0.symbol} / ${p.token1.symbol}`}</Text>
                       </Text>
-                      <SaveIcon
+                      <GreyBadge ml="10px" style={{ fontSize: 14 }}>
+                        {feeTierPercent(p.feeTier)}
+                      </GreyBadge>
+                      {/* <SaveIcon
                         id="watchlist-icon"
                         style={{ marginLeft: '10px' }}
                         fill={savedPools.includes(p.address)}
                         onClick={(e) => {
                           e.stopPropagation()
                           addSavedPool(p.address)
                         }}
```

Comment:
same here

Reply:
We may implement it after the info page is well-tuned